week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -59,8 +99,48 @@ function exerciseTwo(shopping) {
 **/
 function exerciseThree(books) {
   //Write your code in here
+  let bookTitle = books.map(book => `${book.title}: ${book.author}`);
+  let RorN = books.map(read => (read.alreadyRead === true ? `green` : `red`));
+  // console.log(RorNot);
+  let ulEl = document.createElement("ul");
+  let content = document.getElementById("content");
+  content.appendChild(ulEl);
+  ulEl.style.display = "flex"
+  ulEl.style.justifyContent = "space-between";
+  for (let i = 0; i < books.length; i++) {
```

Comment:
The code in here is a bit confusing - could you split it out into functions to make it easier to read?